posydon/grids/psygrid.py

```diff
@@ -2165,14 +2175,24 @@ def say(something):
     say("Inferring which runs to include, from which grid...")
     initial_params = {}
     n_substitutions = 0
+    n_ignored_noRLO = 0
     for grid_index, grid in enumerate(grids):
         for run_index, dir_path in enumerate(grid.MESA_dirs):
             # get the parameters part from the dir name
             params_from_path = initial_values_from_dirname(dir_path)
             if params_from_path in initial_params:
                 n_substitutions += 1
+
             initial_params[params_from_path] = (grid_index, run_index)
-    say("    {} substituions detected.".format(n_substitutions))
+
+            if (grid[run_index].final_values["termination_flag_1"]
+                    == "ignored_no_RLO"):
+                if params_from_path in initial_params:
+                    del initial_params[params_from_path]
+                n_ignored_noRLO += 1
+
+    say("    {} substitutions detected.".format(n_substitutions))
+    say("    {} runs ignored beacuse of no RLOF.".format(n_ignored_noRLO))
```

Comment:
Just a minor thing: should we have this output in case of a grid not starting at RLO? I'd say it would be better to only output this for grids starting at RLO. Hence, I'd put it behind:
`if (newconfig["start_at_RLO"]):`

Reply:
Done!